app/services/support_interface/work_history_break_export.rb

```diff
@@ -0,0 +1,221 @@
+module SupportInterface
+  class WorkHistoryBreakExport
+    class UnexplainedBreak
+      def initialize(month_range:)
+        @month_range = month_range
+      end
+
+      def start_date
+        @month_range.first.prev_month.in_time_zone
+      end
+
+      def end_date
+        @month_range.last.next_month.in_time_zone
+      end
+
+      def length
+        ((end_date.year * 12) + end_date.month) - ((start_date.year * 12) + start_date.month)
+      end
+    end
+
+    def data_for_export
+      applications = ApplicationForm
+                         .where.not(date_of_birth: nil)
+                         .select(:id, :candidate_id, :submitted_at, :date_of_birth, :work_history_completed)
+                         .includes(:application_qualifications, :application_work_experiences, :application_work_history_breaks, :application_choices)
+                         .includes(:application_qualifications, :application_work_experiences, :application_volunteering_experiences, :application_work_history_breaks, :application_choices)
+                         .order(submitted_at: :desc).uniq(&:candidate_id)
+
+      data_for_export = applications.map do |application_form|
+        volunteering_experiences = application_form.application_volunteering_experiences.sort_by(&:start_date)
+        explained_breaks = application_form.application_work_history_breaks.sort_by(&:start_date)
+        unexplained_breaks = get_unexplained_breaks(application_form)
+
+        next if unexplained_breaks.nil? && explained_breaks.nil?
+
+        output = {
+          'Candidate id' => application_form.candidate_id,
+          'Application id' => application_form.id,
+          'Application submitted' => application_form.submitted_at.to_date,
+          'Course choice statuses' => application_form.application_choices.map(&:status).sort,
+          'Start of working life' => start_of_working_life(application_form),
+          'Total time in employment (months)' => total_time_in_employment(application_form),
+        }
+
+        if explained_breaks.any?
```

Comment:
Potentially lines 45 - 54 could be extracted to a private method in order to reduce the size of the `data_for_export` method